### ISSUE TITLE
Upgrade to Amazon Linux 2?

### ISSUE THREAD

**Issue by willsmanley** [FoxxMD/laravel-elasticbeanstalk-queue-worker#13]
Firstly thanks for the great work @FoxxMD! I have enjoyed this as a starting point for my .ebextensions directory on Amazon Linux AMI.

Now since I am upgrading to PHP 7.4, I need to also upgrade to Amazon Linux 2.

It looks like they have strangely removed the very handy "get-config" tool that you utilize in `99deployWorker.config`:
```
command: /opt/elasticbeanstalk/bin/get-config environment > jsonEnv
```

Have you moved to AL2 yourself or do you plan to do so?

If not, is there another way to work around the supervisor deployment without get-config?

**Comment by FoxxMD** [comment]
I'm sorry I am not using the new AMI yet. If you do find a similar tool please let me know!

**Comment by winkelco** [comment]
hello. I have this problem: /usr/bin/env: bash: No such file or directory.

**Comment by MbahDjoe** [comment]
closed due to release of [1.0.0](https://github.com/FoxxMD/laravel-elasticbeanstalk-queue-worker/releases/tag/1.0.0) #14
